pkgs/development/libraries/thrift/0.10.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, boost, zlib, libevent, openssl, python, pkgconfig, bison
+, flex, twisted
+}:
+
+stdenv.mkDerivation rec {
+  name = "thrift-${version}";
+  version = "0.10.0";
+
+  src = fetchurl {
+    url = "https://archive.apache.org/dist/thrift/${version}/${name}.tar.gz";
+    sha256 = "02x1xw0l669idkn6xww39j60kqxzcbmim4mvpb5h9nz8wqnx1292";
+  };
+
+  #enableParallelBuilding = true; problems on hydra
+
+  # Workaround to make the python wrapper not drop this package:
+  # pythonFull.buildEnv.override { extraLibs = [ thrift ]; }
+  pythonPath = [];
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    boost zlib libevent openssl python bison flex twisted
```

Comment:
Are `bison` and `flex` actually used after building the application? My guess is they belong in `nativeBuildInputs`.

Reply:
I'm following the example of the original `thrift` package in version 0.10 here.

Happy to improve that package, but maybe in a separate PR?

Reply:
Separate PR sounds good. Thanks for flipping the extra mile on that!